templates/typescript_gapic/system-test/install.ts.njk

```diff
@@ -17,40 +17,21 @@ limitations under the License.
 -#}
 {% import "../_license.njk" as license -%}
 {{license.license()}}
-import * as execa from 'execa';
-import * as mv from 'mv';
-import {ncp} from 'ncp';
-import * as tmp from 'tmp';
-import {promisify} from 'util';
-
-const keep = false;
-const mvp = promisify(mv);
-const ncpp = promisify(ncp);
-const stagingDir = tmp.dirSync({keep, unsafeCleanup: true});
-const stagingPath = stagingDir.name;
-const pkg = require('../../package.json');
-
-describe('📦 pack and install', () => {
-  it('should be able use the library from a TypeScript application', async function() {
+import {packNTest} from 'pack-n-play';
+import { readFileSync } from 'fs';
+
+describe('typescript consumer tests', () => {
+
+  it('should have correct type signature for {{ api.naming.name }} client library', async function() {
     this.timeout(300000);
-    await execa('npm', ['pack', '--unsafe-perm']);
-    const packageName = pkg.name.replace('@', '').replace('/', '-');
-    const tarball = `${packageName}-${pkg.version}.tgz`;
-    await mvp(tarball, `${stagingPath}/{{ api.naming.productName.toKebabCase() }}.tgz`);
-    await ncpp('system-test/fixtures/sample', `${stagingPath}/`);
-    await execa('npm', ['install', '--unsafe-perm'], {
-      cwd: `${stagingPath}/`,
-      stdio: 'inherit',
-    });
-    await execa('node', ['--throw-deprecation', 'build/src/index.js'], {
-      cwd: `${stagingPath}/`,
-      stdio: 'inherit',
-    });
+    const options = {
+      packageDir: process.cwd(),  // path to your module.
+      sample: {
+        description: 'typescript based used can use my type definitions',
+        ts: readFileSync('./system-test/fixtures/sample/src/index.ts').toString()
```

Comment:
if it's that simple, can we have a JS user version too? I think `pack-n-play` supports it.

Reply:
yes sure, turns out the `ts` or `js` field accepts `string`